src/libproc_macro/lib.rs

```diff
@@ -54,7 +54,7 @@ use syntax::ptr::P;
 /// time!
 #[stable(feature = "proc_macro_lib", since = "1.15.0")]
 pub struct TokenStream {
-    inner: Vec<P<ast::Item>>,
+    inner: TokenStream_,
```

Comment:
Yeah, but since this is a private import of `tokenstream::TokenStream` (as opposed to a type specifically for this purpose) I don't think it's that big of a deal.